Error building firmware image - OpenWrt One

```
{
    "url": "https://sysupgrade.openwrt.org",
    "branch": "SNAPSHOT",
    "revision": "r27700-170ecbecfc",
    "efi": null,
    "advanced_mode": "1",
    "request_hash": "8c140f24fa43c4cbbab2e10082c3c27b",
    "sha256_unsigned": "",
    "client": "luci/24.205.37920~e55f9d9",
    "packages": {
        "kmod-usb-storage": "6.6.54-r1",
        "iwinfo": "2024.10.01~714e4199-r1",
        "luci-app-nlbwmon": "24.148.86114~d174bb8",
        "opkg": "2022.02.24~d038e5b6-r2",
        "libpthread": "1.2.5-r4",
        "busybox": "1.36.1-r2",
        "f2fs-tools": "1.16.0-r3",
        "libiwinfo": "2024.10.01~714e4199-r1",
        "kmod-nft-fib": "6.6.54-r1",
        "kmod-nfnetlink": "6.6.54-r1",
        "kmod-nf-reject6": "6.6.54-r1",
        "kmod-crypto-aead": "6.6.54-r1",
        "luci-app-https-dns-proxy": "2023.12.26-1",
        "kmod-nf-flow": "6.6.54-r1",
        "kmod-lib-crc-ccitt": "6.6.54-r1",
        "getrandom": "2024.04.26~85f10530-r1",
        "kmod-phy-aquantia": "6.6.54-r1",
        "kmod-pppoe": "6.6.54-r1",
        "kmod-pppox": "6.6.54-r1",
        "kmod-nf-reject": "6.6.54-r1",
        "procd-ujail": "2024.07.07~f230c117-r1",
        "base-files": "1625~170ecbecfc",
        "kmod-nf-nat": "6.6.54-r1",
        "kmod-crypto-crc32c": "6.6.54-r1",
        "libustream-mbedtls": "2024.07.28~99bd3d2b-r1",
        "fitblk": "2",
        "firewall4": "2024.05.21~4c01d1eb-r1",
        "uboot-envtools": "2024.07-r1",
        "dnsmasq": "2.90-r3",
        "procd": "2024.07.07~f230c117-r1",
        "usbutils": "017-r1",
        "kmod-crypto-hmac": "6.6.54-r1",
        "kmod-nvme": "6.6.54-r1",
        "kmod-crypto-acompress": "6.6.54-r1",
        "kmod-lib-crc32c": "6.6.54-r1",
        "kmod-rtc-pcf8563": "6.6.54-r1",
        "luci-app-usteer": "24.233.60536~067f1c5",
        "kmod-nft-nat": "6.6.54-r1",
        "kmod-usb3": "6.6.54-r1",
        "kmod-fs-exfat": "6.6.54-r1",
        "libmbedtls": "3.6.1-r1",
        "libmnl": "1.0.5-r1",
        "jansson": "2.14-r3",
        "odhcp6c": "2023.05.12~bcd28363-r20",
        "uclient-fetch": "2024.04.19~e8780fa7-r1",
        "uci": "2023.08.10~5781664d-r1",
        "kmod-fs-ext4": "6.6.54-r1",
        "dropbear": "2024.85-r1",
        "curl": "8.10.1-r1",
        "kmod-hwmon-core": "6.6.54-r1",
        "libnftnl": "1.2.6-r1",
        "mtd": "26",
        "odhcpd-ipv6only": "2024.05.08~a2988231-r1",
        "procd-seccomp": "2024.07.07~f230c117-r1",
        "kmod-crypto-hw-safexcel": "6.6.54-r1",
        "libjson-script": "2024.03.29~eb9bcb64-r1",
        "kmod-crypto-authenc": "6.6.54-r1",
        "kmod-mt7915e": "6.6.54.2024.09.29~680bc70f-r1",
        "kmod-nf-log": "6.6.54-r1",
        "urandom-seed": "3",
        "blockd": "2024.07.14~408c2cc4-r1",
        "ppp": "2.4.9_git20210104-r6",
        "kmod-leds-gpio": "6.6.54-r1",
        "kmod-gpio-button-hotplug": "6.6.54-r4",
        "logd": "2024.04.26~85f10530-r1",
        "kmod-nf-log6": "6.6.54-r1",
        "luci-app-attendedsysupgrade": "24.205.37920~e55f9d9",
        "kmod-libphy": "6.6.54-r1",
        "kmod-crypto-des": "6.6.54-r1",
        "e2fsprogs": "1.47.0-r2",
        "kmod-ppp": "6.6.54-r1",
        "kmod-nft-offload": "6.6.54-r1",
        "kmod-fs-f2fs": "6.6.54-r1",
        "wpad-mbedtls": "2024.09.15~5ace39b0-r1",
        "mt7981-wo-firmware": "20240909-r1",
        "kmod-nf-conntrack": "6.6.54-r1",
        "ca-bundle": "20240203-r1",
        "libuclient": "2024.04.19~e8780fa7-r1",
        "luci": "23.051.66410~a505bb1",
        "kmod-nf-conntrack6": "6.6.54-r1",
        "kmod-mt7981-firmware": "6.6.54.2024.09.29~680bc70f-r1",
        "kmod-lib-lzo": "6.6.54-r1",
        "kmod-phy-airoha-en8811h": "6.6.54-r1",
        "kmod-nft-core": "6.6.54-r1",
        "hostapd-common": "2024.09.15~5ace39b0-r1",
        "gdisk": "1.0.10-r1",
        "urngd": "2023.11.01~44365eb1-r1",
        "kmod-slhc": "6.6.54-r1",
        "ppp-mod-pppoe": "2.4.9_git20210104-r6"
    },
    "profile": "openwrt,one",
    "target": "mediatek/filogic",
    "version": "SNAPSHOT",
    "diff_packages": true,
    "filesystem": "squashfs"
}

STDOUT:

Checking 'true'... ok.
Checking 'false'... ok.
Checking 'working-make'... ok.
Checking 'case-sensitive-fs'... ok.
Checking 'proper-umask'... ok.
Checking 'perl-data-dumper'... ok.
Checking 'perl-findbin'... ok.
Checking 'perl-file-copy'... ok.
Checking 'perl-file-compare'... ok.
Checking 'perl-thread-queue'... ok.
Checking 'perl-ipc-cmd'... ok.
Checking 'tar'... ok.
Checking 'find'... ok.
Checking 'bash'... ok.
Checking 'xargs'... ok.
Checking 'patch'... ok.
Checking 'diff'... ok.
Checking 'cp'... ok.
Checking 'seq'... ok.
Checking 'awk'... ok.
Checking 'grep'... ok.
Checking 'egrep'... ok.
Checking 'getopt'... ok.
Checking 'realpath'... ok.
Checking 'stat'... ok.
Checking 'gzip'... ok.
Checking 'unzip'... ok.
Checking 'bzip2'... ok.
Checking 'wget'... ok.
Checking 'install'... ok.
Checking 'perl'... ok.
Checking 'python'... ok.
Checking 'python3'... ok.
Checking 'python3-distutils'... ok.
Checking 'python3-stdlib'... ok.
Checking 'file'... ok.
Checking 'which'... ok.
Checking 'ldconfig-stub'... ok.
Building images for mediatek - OpenWrt One
Packages: base-files blockd busybox ca-bundle curl dnsmasq dropbear e2fsprogs f2fs-tools firewall4 fitblk gdisk getrandom hostapd-common iwinfo jansson kmod-crypto-acompress kmod-crypto-aead kmod-crypto-authenc kmod-crypto-crc32c kmod-crypto-des kmod-crypto-hmac kmod-crypto-hw-safexcel kmod-fs-exfat kmod-fs-ext4 kmod-fs-f2fs kmod-gpio-button-hotplug kmod-hwmon-core kmod-leds-gpio kmod-lib-crc-ccitt kmod-lib-crc32c kmod-lib-lzo kmod-libphy kmod-mt7915e kmod-mt7981-firmware kmod-nf-conntrack kmod-nf-conntrack6 kmod-nf-flow kmod-nf-log kmod-nf-log6 kmod-nf-nat kmod-nf-reject kmod-nf-reject6 kmod-nfnetlink kmod-nft-core kmod-nft-fib kmod-nft-nat kmod-nft-offload kmod-nvme kmod-phy-airoha-en8811h kmod-phy-aquantia kmod-ppp kmod-pppoe kmod-pppox kmod-rtc-pcf8563 kmod-slhc kmod-usb-storage kmod-usb3 libiwinfo libjson-script libmbedtls libmnl libnftnl libpthread libuclient libustream-mbedtls logd luci luci-app-attendedsysupgrade luci-app-https-dns-proxy luci-app-nlbwmon luci-app-usteer mt7981-wo-firmware mtd odhcp6c odhcpd-ipv6only opkg ppp ppp-mod-pppoe procd procd-seccomp procd-ujail uboot-envtools uci uclient-fetch urandom-seed urngd usbutils wpad-mbedtls base-files busybox ca-bundle dnsmasq dropbear firewall4 fitblk kernel kmod-crypto-hw-safexcel kmod-gpio-button-hotplug kmod-leds-gpio kmod-mt7915e kmod-mt7981-firmware kmod-nft-offload kmod-nvme kmod-phy-airoha-en8811h kmod-phy-aquantia kmod-rtc-pcf8563 kmod-usb3 libustream-mbedtls logd mt7981-wo-firmware mtd odhcp6c odhcpd-ipv6only opkg ppp ppp-mod-pppoe procd procd-seccomp procd-ujail uboot-envtools uci uclient-fetch urandom-seed urngd


Installing packages...
Installing libc (1.2.5-r4) to root...
Installing libgcc1 (13.3.0-r4) to root...
Copying /builder/dl/libgcc1_13.3.0-r4_aarch64_cortex-a53.ipk.
Configuring libgcc1.
Configuring libc.
Installing kernel (6.6.56~fd58c1f219e8bdd2c94a26335acdfb0a-r1) to root...
Configuring kernel.
Installing base-files (1627~f9f2dd1d93) to root...
Copying /builder/dl/base-files_1627~f9f2dd1d93_aarch64_cortex-a53.ipk.
Installing libubox20240329 (2024.03.29~eb9bcb64-r1) to root...
Copying /builder/dl/libubox20240329_2024.03.29~eb9bcb64-r1_aarch64_cortex-a53.ipk.
Installing libuci20130104 (2023.08.10~5781664d-r1) to root...
Copying /builder/dl/libuci20130104_2023.08.10~5781664d-r1_aarch64_cortex-a53.ipk.
Installing libnl-tiny1 (2023.12.05~965c4bf4-r1) to root...
Copying /builder/dl/libnl-tiny1_2023.12.05~965c4bf4-r1_aarch64_cortex-a53.ipk.
Installing libubus20231128 (2023.11.28~f84eb599-r1) to root...
Copying /builder/dl/libubus20231128_2023.11.28~f84eb599-r1_aarch64_cortex-a53.ipk.
Installing libjson-c5 (0.18-r1) to root...
Copying /builder/dl/libjson-c5_0.18-r1_aarch64_cortex-a53.ipk.
Installing libblobmsg-json20240329 (2024.03.29~eb9bcb64-r1) to root...
Copying /builder/dl/libblobmsg-json20240329_2024.03.29~eb9bcb64-r1_aarch64_cortex-a53.ipk.
Installing ubusd (2023.11.28~f84eb599-r1) to root...
Copying /builder/dl/ubusd_2023.11.28~f84eb599-r1_aarch64_cortex-a53.ipk.
Installing ubus (2023.11.28~f84eb599-r1) to root...
Copying /builder/dl/ubus_2023.11.28~f84eb599-r1_aarch64_cortex-a53.ipk.
Installing jshn (2024.03.29~eb9bcb64-r1) to root...
Copying /builder/dl/jshn_2024.03.29~eb9bcb64-r1_aarch64_cortex-a53.ipk.
Installing libudebug (2023.12.06~6d3f51f9) to root...
Copying /builder/dl/libudebug_2023.12.06~6d3f51f9_aarch64_cortex-a53.ipk.
Installing libucode20230711 (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/libucode20230711_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing ucode (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing ucode-mod-fs (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-fs_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing netifd (2024.10.06~3c6265fd-r1) to root...
Copying /builder/dl/netifd_2024.10.06~3c6265fd-r1_aarch64_cortex-a53.ipk.
Installing jsonfilter (2024.01.23~594cfa86-r1) to root...
Copying /builder/dl/jsonfilter_2024.01.23~594cfa86-r1_aarch64_cortex-a53.ipk.
Installing usign (2020.05.23~f1f65026-r1) to root...
Copying /builder/dl/usign_2020.05.23~f1f65026-r1_aarch64_cortex-a53.ipk.
Installing openwrt-keyring (2024.09.28~3c066ce2-r1) to root...
Copying /builder/dl/openwrt-keyring_2024.09.28~3c066ce2-r1_aarch64_cortex-a53.ipk.
Installing ubi-utils (2.2.1-r1) to root...
Copying /builder/dl/ubi-utils_2.2.1-r1_aarch64_cortex-a53.ipk.
Installing ubox (2024.04.26~85f10530-r1) to root...
Copying /builder/dl/ubox_2024.04.26~85f10530-r1_aarch64_cortex-a53.ipk.
Installing fstools (2024.07.14~408c2cc4-r1) to root...
Copying /builder/dl/fstools_2024.07.14~408c2cc4-r1_aarch64_cortex-a53.ipk.
Installing fwtool (2019.11.12~8f7fe925-r1) to root...
Copying /builder/dl/fwtool_2019.11.12~8f7fe925-r1_aarch64_cortex-a53.ipk.
Installing blockd (2024.07.14~408c2cc4-r1) to root...
Copying /builder/dl/blockd_2024.07.14~408c2cc4-r1_aarch64_cortex-a53.ipk.
Installing block-mount (2024.07.14~408c2cc4-r1) to root...
Copying /builder/dl/block-mount_2024.07.14~408c2cc4-r1_aarch64_cortex-a53.ipk.
Installing kmod-fs-autofs4 (6.6.56-r1) to root...
Copying /builder/dl/kmod-fs-autofs4_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing busybox (1.36.1-r2) to root...
Copying /builder/dl/busybox_1.36.1-r2_aarch64_cortex-a53.ipk.
Installing ca-bundle (20240203-r1) to root...
Copying /builder/dl/ca-bundle_20240203-r1_all.ipk.
Installing curl (8.10.1-r1) to root...
Copying /builder/dl/curl_8.10.1-r1_aarch64_cortex-a53.ipk.
Installing libmbedtls21 (3.6.1-r1) to root...
Copying /builder/dl/libmbedtls21_3.6.1-r1_aarch64_cortex-a53.ipk.
Installing libnghttp2-14 (1.63.0-r1) to root...
Copying /builder/dl/libnghttp2-14_1.63.0-r1_aarch64_cortex-a53.ipk.
Installing libcurl4 (8.10.1-r1) to root...
Copying /builder/dl/libcurl4_8.10.1-r1_aarch64_cortex-a53.ipk.
Installing dnsmasq (2.90-r3) to root...
Copying /builder/dl/dnsmasq_2.90-r3_aarch64_cortex-a53.ipk.
Installing dropbear (2024.85-r1) to root...
Copying /builder/dl/dropbear_2024.85-r1_aarch64_cortex-a53.ipk.
Installing e2fsprogs (1.47.0-r2) to root...
Copying /builder/dl/e2fsprogs_1.47.0-r2_aarch64_cortex-a53.ipk.
Installing libpthread (1.2.5-r4) to root...
Copying /builder/dl/libpthread_1.2.5-r4_aarch64_cortex-a53.ipk.
Installing librt (1.2.5-r4) to root...
Copying /builder/dl/librt_1.2.5-r4_aarch64_cortex-a53.ipk.
Installing libuuid1 (2.40.2-r1) to root...
Copying /builder/dl/libuuid1_2.40.2-r1_aarch64_cortex-a53.ipk.
Installing libblkid1 (2.40.2-r1) to root...
Copying /builder/dl/libblkid1_2.40.2-r1_aarch64_cortex-a53.ipk.
Installing libcomerr0 (1.47.0-r2) to root...
Copying /builder/dl/libcomerr0_1.47.0-r2_aarch64_cortex-a53.ipk.
Installing libss2 (1.47.0-r2) to root...
Copying /builder/dl/libss2_1.47.0-r2_aarch64_cortex-a53.ipk.
Installing libext2fs2 (1.47.0-r2) to root...
Copying /builder/dl/libext2fs2_1.47.0-r2_aarch64_cortex-a53.ipk.
Installing libe2p2 (1.47.0-r2) to root...
Copying /builder/dl/libe2p2_1.47.0-r2_aarch64_cortex-a53.ipk.
Installing f2fs-tools (1.16.0-r3) to root...
Copying /builder/dl/f2fs-tools_1.16.0-r3_aarch64_cortex-a53.ipk.
Installing libf2fs6 (1.16.0-r3) to root...
Copying /builder/dl/libf2fs6_1.16.0-r3_aarch64_cortex-a53.ipk.
Installing mkf2fs (1.16.0-r3) to root...
Copying /builder/dl/mkf2fs_1.16.0-r3_aarch64_cortex-a53.ipk.
Installing f2fsck (1.16.0-r3) to root...
Copying /builder/dl/f2fsck_1.16.0-r3_aarch64_cortex-a53.ipk.
Installing firewall4 (2024.05.21~4c01d1eb-r1) to root...
Copying /builder/dl/firewall4_2024.05.21~4c01d1eb-r1_aarch64_cortex-a53.ipk.
Installing kmod-nfnetlink (6.6.56-r1) to root...
Copying /builder/dl/kmod-nfnetlink_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-reject (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-reject_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-reject6 (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-reject6_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-conntrack (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-conntrack_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-conntrack6 (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-conntrack6_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-nat (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-nat_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-log (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-log_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-log6 (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-log6_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-hash (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-hash_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-crc32c (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-crc32c_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-lib-crc32c (6.6.56-r1) to root...
Copying /builder/dl/kmod-lib-crc32c_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nft-core (6.6.56-r1) to root...
Copying /builder/dl/kmod-nft-core_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nft-fib (6.6.56-r1) to root...
Copying /builder/dl/kmod-nft-fib_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-flow (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-flow_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nft-nat (6.6.56-r1) to root...
Copying /builder/dl/kmod-nft-nat_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nft-offload (6.6.56-r1) to root...
Copying /builder/dl/kmod-nft-offload_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing libmnl0 (1.0.5-r1) to root...
Copying /builder/dl/libmnl0_1.0.5-r1_aarch64_cortex-a53.ipk.
Installing libnftnl11 (1.2.6-r1) to root...
Copying /builder/dl/libnftnl11_1.2.6-r1_aarch64_cortex-a53.ipk.
Installing jansson4 (2.14-r3) to root...
Copying /builder/dl/jansson4_2.14-r3_aarch64_cortex-a53.ipk.
Installing nftables-json (1.0.9-r1) to root...
Copying /builder/dl/nftables-json_1.0.9-r1_aarch64_cortex-a53.ipk.
Installing ucode-mod-ubus (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-ubus_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing ucode-mod-uci (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-uci_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing fitblk (2) to root...
Copying /builder/dl/fitblk_2_aarch64_cortex-a53.ipk.
Installing gdisk (1.0.10-r1) to root...
Copying /builder/dl/gdisk_1.0.10-r1_aarch64_cortex-a53.ipk.
Installing libstdcpp6 (13.3.0-r4) to root...
Copying /builder/dl/libstdcpp6_13.3.0-r4_aarch64_cortex-a53.ipk.
Installing getrandom (2024.04.26~85f10530-r1) to root...
Copying /builder/dl/getrandom_2024.04.26~85f10530-r1_aarch64_cortex-a53.ipk.
Installing hostapd-common (2024.09.15~5ace39b0-r1) to root...
Copying /builder/dl/hostapd-common_2024.09.15~5ace39b0-r1_aarch64_cortex-a53.ipk.
Installing iwinfo (2024.10.01~714e4199-r1) to root...
Copying /builder/dl/iwinfo_2024.10.01~714e4199-r1_aarch64_cortex-a53.ipk.
Installing libiwinfo-data (2024.10.01~714e4199-r1) to root...
Copying /builder/dl/libiwinfo-data_2024.10.01~714e4199-r1_aarch64_cortex-a53.ipk.
Installing libiwinfo20230701 (2024.10.01~714e4199-r1) to root...
Copying /builder/dl/libiwinfo20230701_2024.10.01~714e4199-r1_aarch64_cortex-a53.ipk.
Package jansson4 (2.14-r3) installed in root is up to date.
Installing kmod-crypto-acompress (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-acompress_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-aead (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-aead_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-null (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-null_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-authenc (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-authenc_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-manager (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-manager_6.6.56-r1_aarch64_cortex-a53.ipk.
Package kmod-crypto-crc32c (6.6.56-r1) installed in root is up to date.
Installing kmod-crypto-des (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-des_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-hmac (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-hmac_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-hw-safexcel (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-hw-safexcel_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing eip197-mini-firmware (20240909-r1) to root...
Copying /builder/dl/eip197-mini-firmware_20240909-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-md5 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-md5_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-sha1 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-sha1_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-sha256 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-sha256_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-sha512 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-sha512_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-fs-exfat (6.6.56-r1) to root...
Copying /builder/dl/kmod-fs-exfat_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-nls-base (6.6.56-r1) to root...
Copying /builder/dl/kmod-nls-base_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-fs-ext4 (6.6.56-r1) to root...
Copying /builder/dl/kmod-fs-ext4_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-lib-crc16 (6.6.56-r1) to root...
Copying /builder/dl/kmod-lib-crc16_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-fs-f2fs (6.6.56-r1) to root...
Copying /builder/dl/kmod-fs-f2fs_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-crc32 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-crc32_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-gpio-button-hotplug (6.6.56-r5) to root...
Copying /builder/dl/kmod-gpio-button-hotplug_6.6.56-r5_aarch64_cortex-a53.ipk.
Installing kmod-hwmon-core (6.6.56-r1) to root...
Copying /builder/dl/kmod-hwmon-core_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-leds-gpio (6.6.56-r1) to root...
Copying /builder/dl/kmod-leds-gpio_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-lib-crc-ccitt (6.6.56-r1) to root...
Copying /builder/dl/kmod-lib-crc-ccitt_6.6.56-r1_aarch64_cortex-a53.ipk.
Package kmod-lib-crc32c (6.6.56-r1) installed in root is up to date.
Installing kmod-lib-lzo (6.6.56-r1) to root...
Copying /builder/dl/kmod-lib-lzo_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-libphy (6.6.56-r1) to root...
Copying /builder/dl/kmod-libphy_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-mt7915e (6.6.56.2024.10.11.1~ecca0e77-r1) to root...
Copying /builder/dl/kmod-mt7915e_6.6.56.2024.10.11.1~ecca0e77-r1_aarch64_cortex-a53.ipk.
Installing iw (6.9-r1) to root...
Copying /builder/dl/iw_6.9-r1_aarch64_cortex-a53.ipk.
Installing ucode-mod-nl80211 (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-nl80211_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing ucode-mod-rtnl (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-rtnl_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing wifi-scripts (1.0-r1) to root...
Copying /builder/dl/wifi-scripts_1.0-r1_all.ipk.
Installing wireless-regdb (2024.10.07-r1) to root...
Copying /builder/dl/wireless-regdb_2024.10.07-r1_all.ipk.
Installing kmod-cfg80211 (6.6.56.6.11.2-r1) to root...
Copying /builder/dl/kmod-cfg80211_6.6.56.6.11.2-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-cmac (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-cmac_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-sha3 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-sha3_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-rng (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-rng_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-geniv (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-geniv_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-seqiv (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-seqiv_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-ctr (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-ctr_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-ccm (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-ccm_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-gf128 (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-gf128_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-ghash (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-ghash_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-crypto-gcm (6.6.56-r1) to root...
Copying /builder/dl/kmod-crypto-gcm_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-mac80211 (6.6.56.6.11.2-r1) to root...
Copying /builder/dl/kmod-mac80211_6.6.56.6.11.2-r1_aarch64_cortex-a53.ipk.
Installing kmod-mt76-core (6.6.56.2024.10.11.1~ecca0e77-r1) to root...
Copying /builder/dl/kmod-mt76-core_6.6.56.2024.10.11.1~ecca0e77-r1_aarch64_cortex-a53.ipk.
Installing kmod-mt76-connac (6.6.56.2024.10.11.1~ecca0e77-r1) to root...
Copying /builder/dl/kmod-mt76-connac_6.6.56.2024.10.11.1~ecca0e77-r1_aarch64_cortex-a53.ipk.
Installing kmod-thermal (6.6.56-r1) to root...
Copying /builder/dl/kmod-thermal_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-mt7981-firmware (6.6.56.2024.10.11.1~ecca0e77-r1) to root...
Copying /builder/dl/kmod-mt7981-firmware_6.6.56.2024.10.11.1~ecca0e77-r1_aarch64_cortex-a53.ipk.
Package kmod-nf-conntrack (6.6.56-r1) installed in root is up to date.
Package kmod-nf-conntrack6 (6.6.56-r1) installed in root is up to date.
Package kmod-nf-flow (6.6.56-r1) installed in root is up to date.
Package kmod-nf-log (6.6.56-r1) installed in root is up to date.
Package kmod-nf-log6 (6.6.56-r1) installed in root is up to date.
Package kmod-nf-nat (6.6.56-r1) installed in root is up to date.
Package kmod-nf-reject (6.6.56-r1) installed in root is up to date.
Package kmod-nf-reject6 (6.6.56-r1) installed in root is up to date.
Package kmod-nfnetlink (6.6.56-r1) installed in root is up to date.
Package kmod-nft-core (6.6.56-r1) installed in root is up to date.
Package kmod-nft-fib (6.6.56-r1) installed in root is up to date.
Package kmod-nft-nat (6.6.56-r1) installed in root is up to date.
Package kmod-nft-offload (6.6.56-r1) installed in root is up to date.
Installing kmod-nvme (6.6.56-r1) to root...
Copying /builder/dl/kmod-nvme_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-phy-airoha-en8811h (6.6.56-r1) to root...
Copying /builder/dl/kmod-phy-airoha-en8811h_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing airoha-en8811h-firmware (20240909-r1) to root...
Copying /builder/dl/airoha-en8811h-firmware_20240909-r1_aarch64_cortex-a53.ipk.
Installing kmod-phy-aquantia (6.6.56-r1) to root...
Copying /builder/dl/kmod-phy-aquantia_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-ppp (6.6.56-r1) to root...
Copying /builder/dl/kmod-ppp_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-slhc (6.6.56-r1) to root...
Copying /builder/dl/kmod-slhc_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-pppoe (6.6.56-r1) to root...
Copying /builder/dl/kmod-pppoe_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-pppox (6.6.56-r1) to root...
Copying /builder/dl/kmod-pppox_6.6.56-r1_aarch64_cortex-a53.ipk.
Package kmod-pppox (6.6.56-r1) installed in root is up to date.
Installing kmod-rtc-pcf8563 (6.6.56-r1) to root...
Copying /builder/dl/kmod-rtc-pcf8563_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-i2c-core (6.6.56-r1) to root...
Copying /builder/dl/kmod-i2c-core_6.6.56-r1_aarch64_cortex-a53.ipk.
Package kmod-slhc (6.6.56-r1) installed in root is up to date.
Installing kmod-usb-storage (6.6.56-r1) to root...
Copying /builder/dl/kmod-usb-storage_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-scsi-core (6.6.56-r1) to root...
Copying /builder/dl/kmod-scsi-core_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-usb-core (6.6.56-r1) to root...
Copying /builder/dl/kmod-usb-core_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-usb3 (6.6.56-r1) to root...
Copying /builder/dl/kmod-usb3_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-usb-xhci-hcd (6.6.56-r1) to root...
Copying /builder/dl/kmod-usb-xhci-hcd_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing kmod-usb-xhci-mtk (6.6.56-r1) to root...
Copying /builder/dl/kmod-usb-xhci-mtk_6.6.56-r1_aarch64_cortex-a53.ipk.
Package libiwinfo20230701 (2024.10.01~714e4199-r1) installed in root is up to date.
Installing libjson-script20240329 (2024.03.29~eb9bcb64-r1) to root...
Copying /builder/dl/libjson-script20240329_2024.03.29~eb9bcb64-r1_aarch64_cortex-a53.ipk.
Package libmbedtls21 (3.6.1-r1) installed in root is up to date.
Package libmnl0 (1.0.5-r1) installed in root is up to date.
Package libnftnl11 (1.2.6-r1) installed in root is up to date.
Package libpthread (1.2.5-r4) installed in root is up to date.
Installing libuclient20201210 (2024.04.19~e8780fa7-r1) to root...
Copying /builder/dl/libuclient20201210_2024.04.19~e8780fa7-r1_aarch64_cortex-a53.ipk.
Installing libustream-mbedtls20201210 (2024.07.28~99bd3d2b-r1) to root...
Copying /builder/dl/libustream-mbedtls20201210_2024.07.28~99bd3d2b-r1_aarch64_cortex-a53.ipk.
Installing logd (2024.04.26~85f10530-r1) to root...
Copying /builder/dl/logd_2024.04.26~85f10530-r1_aarch64_cortex-a53.ipk.
Installing luci (23.051.66410~a505bb1) to root...
Copying /builder/dl/luci_23.051.66410~a505bb1_all.ipk.
Installing luci-proto-ipv6 (24.148.86114~fbbfdb4) to root...
Copying /builder/dl/luci-proto-ipv6_24.148.86114~fbbfdb4_all.ipk.
Installing rpcd (2024.09.17~9f4b86e7-r1) to root...
Copying /builder/dl/rpcd_2024.09.17~9f4b86e7-r1_aarch64_cortex-a53.ipk.
Installing rpcd-mod-file (2024.09.17~9f4b86e7-r1) to root...
Copying /builder/dl/rpcd-mod-file_2024.09.17~9f4b86e7-r1_aarch64_cortex-a53.ipk.
Installing rpcd-mod-luci (20240305-r1) to root...
Copying /builder/dl/rpcd-mod-luci_20240305-r1_aarch64_cortex-a53.ipk.
Installing rpcd-mod-ucode (2024.09.17~9f4b86e7-r1) to root...
Copying /builder/dl/rpcd-mod-ucode_2024.09.17~9f4b86e7-r1_aarch64_cortex-a53.ipk.
Installing cgi-io (2022.08.10~901b0f04-r21) to root...
Copying /builder/dl/cgi-io_2022.08.10~901b0f04-r21_aarch64_cortex-a53.ipk.
Installing ucode-mod-math (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-math_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Installing ucode-mod-html (1) to root...
Copying /builder/dl/ucode-mod-html_1_aarch64_cortex-a53.ipk.
Installing liblucihttp0 (2023.03.15~9b5b683f-r1) to root...
Copying /builder/dl/liblucihttp0_2023.03.15~9b5b683f-r1_aarch64_cortex-a53.ipk.
Installing liblucihttp-ucode (2023.03.15~9b5b683f-r1) to root...
Copying /builder/dl/liblucihttp-ucode_2023.03.15~9b5b683f-r1_aarch64_cortex-a53.ipk.
Installing luci-base (24.289.85812~4afb477) to root...
Copying /builder/dl/luci-base_24.289.85812~4afb477_aarch64_cortex-a53.ipk.
Installing luci-app-firewall (24.148.86114~fbbfdb4) to root...
Copying /builder/dl/luci-app-firewall_24.148.86114~fbbfdb4_all.ipk.
Installing rpcd-mod-iwinfo (2024.09.17~9f4b86e7-r1) to root...
Copying /builder/dl/rpcd-mod-iwinfo_2024.09.17~9f4b86e7-r1_aarch64_cortex-a53.ipk.
Installing luci-mod-status (24.262.77075~dfb8670) to root...
Copying /builder/dl/luci-mod-status_24.262.77075~dfb8670_aarch64_cortex-a53.ipk.
Installing luci-mod-system (24.165.47464~37ea60d) to root...
Copying /builder/dl/luci-mod-system_24.165.47464~37ea60d_all.ipk.
Installing luci-mod-network (24.281.62975~383edb4) to root...
Copying /builder/dl/luci-mod-network_24.281.62975~383edb4_all.ipk.
Installing luci-mod-admin-full (19.253.48496~3f93650) to root...
Copying /builder/dl/luci-mod-admin-full_19.253.48496~3f93650_all.ipk.
Installing luci-proto-ppp (24.249.84482~02b58b8) to root...
Copying /builder/dl/luci-proto-ppp_24.249.84482~02b58b8_all.ipk.
Installing luci-theme-bootstrap (24.284.01692~c68d5c8) to root...
Copying /builder/dl/luci-theme-bootstrap_24.284.01692~c68d5c8_all.ipk.
Installing rpcd-mod-rrdns (20170710) to root...
Copying /builder/dl/rpcd-mod-rrdns_20170710_aarch64_cortex-a53.ipk.
Installing uhttpd (2023.06.25~34a8a74d-r4) to root...
Copying /builder/dl/uhttpd_2023.06.25~34a8a74d-r4_aarch64_cortex-a53.ipk.
Installing uhttpd-mod-ubus (2023.06.25~34a8a74d-r4) to root...
Copying /builder/dl/uhttpd-mod-ubus_2023.06.25~34a8a74d-r4_aarch64_cortex-a53.ipk.
Installing luci-light (23.024.33244~34dee82) to root...
Copying /builder/dl/luci-light_23.024.33244~34dee82_all.ipk.
Installing uclient-fetch (2024.04.19~e8780fa7-r1) to root...
Copying /builder/dl/uclient-fetch_2024.04.19~e8780fa7-r1_aarch64_cortex-a53.ipk.
Installing opkg (2024.10.16~38eccbb1-r1) to root...
Copying /builder/dl/opkg_2024.10.16~38eccbb1-r1_aarch64_cortex-a53.ipk.
Installing luci-app-opkg (24.191.51591~bff2a43) to root...
Copying /builder/dl/luci-app-opkg_24.191.51591~bff2a43_all.ipk.
Installing luci-app-attendedsysupgrade (24.290.85088~46fb560) to root...
Copying /builder/dl/luci-app-attendedsysupgrade_24.290.85088~46fb560_all.ipk.
Installing rpcd-mod-rpcsys (2024.09.17~9f4b86e7-r1) to root...
Copying /builder/dl/rpcd-mod-rpcsys_2024.09.17~9f4b86e7-r1_aarch64_cortex-a53.ipk.
Installing attendedsysupgrade-common (8) to root...
Copying /builder/dl/attendedsysupgrade-common_8_all.ipk.
Installing luci-app-https-dns-proxy (2023.12.26-1) to root...
Copying /builder/dl/luci-app-https-dns-proxy_2023.12.26-1_all.ipk.
Installing libcares (1.33.1-r1) to root...
Copying /builder/dl/libcares_1.33.1-r1_aarch64_cortex-a53.ipk.
Installing libev (4.33-r2) to root...
Copying /builder/dl/libev_4.33-r2_aarch64_cortex-a53.ipk.
Installing resolveip (2) to root...
Copying /builder/dl/resolveip_2_aarch64_cortex-a53.ipk.
Installing https-dns-proxy (2023.12.26-r1) to root...
Copying /builder/dl/https-dns-proxy_2023.12.26-r1_aarch64_cortex-a53.ipk.
Installing luci-app-nlbwmon (24.148.86114~d174bb8) to root...
Copying /builder/dl/luci-app-nlbwmon_24.148.86114~d174bb8_all.ipk.
Installing luci-lib-chartjs (24.081.35633~10e3c44) to root...
Copying /builder/dl/luci-lib-chartjs_24.081.35633~10e3c44_all.ipk.
Installing zlib (1.3.1-r1) to root...
Copying /builder/dl/zlib_1.3.1-r1_aarch64_cortex-a53.ipk.
Installing kmod-nf-conntrack-netlink (6.6.56-r1) to root...
Copying /builder/dl/kmod-nf-conntrack-netlink_6.6.56-r1_aarch64_cortex-a53.ipk.
Installing nlbwmon (2024.02.21~c7616bcf-r1) to root...
Copying /builder/dl/nlbwmon_2024.02.21~c7616bcf-r1_aarch64_cortex-a53.ipk.
Installing luci-app-usteer (24.233.60536~067f1c5) to root...
Copying /builder/dl/luci-app-usteer_24.233.60536~067f1c5_all.ipk.
Installing usteer (2022.08.18~7d2b17c9-r1) to root...
Copying /builder/dl/usteer_2022.08.18~7d2b17c9-r1_aarch64_cortex-a53.ipk.
Installing mt7981-wo-firmware (20240909-r1) to root...
Copying /builder/dl/mt7981-wo-firmware_20240909-r1_aarch64_cortex-a53.ipk.
Installing mtd (26) to root...
Copying /builder/dl/mtd_26_aarch64_cortex-a53.ipk.
Installing odhcp6c (2023.05.12~bcd28363-r20) to root...
Copying /builder/dl/odhcp6c_2023.05.12~bcd28363-r20_aarch64_cortex-a53.ipk.
Installing odhcpd-ipv6only (2024.05.08~a2988231-r1) to root...
Copying /builder/dl/odhcpd-ipv6only_2024.05.08~a2988231-r1_aarch64_cortex-a53.ipk.
Package opkg (2024.10.16~38eccbb1-r1) installed in root is up to date.
Installing ppp (2.5.1-r1) to root...
Copying /builder/dl/ppp_2.5.1-r1_aarch64_cortex-a53.ipk.
Installing ppp-mod-pppoe (2.5.1-r1) to root...
Copying /builder/dl/ppp-mod-pppoe_2.5.1-r1_aarch64_cortex-a53.ipk.
Installing procd (2024.07.07~f230c117-r1) to root...
Copying /builder/dl/procd_2024.07.07~f230c117-r1_aarch64_cortex-a53.ipk.
Installing procd-seccomp (2024.07.07~f230c117-r1) to root...
Copying /builder/dl/procd-seccomp_2024.07.07~f230c117-r1_aarch64_cortex-a53.ipk.
Installing procd-ujail (2024.07.07~f230c117-r1) to root...
Copying /builder/dl/procd-ujail_2024.07.07~f230c117-r1_aarch64_cortex-a53.ipk.
Installing uboot-envtools (2024.07-r1) to root...
Copying /builder/dl/uboot-envtools_2024.07-r1_aarch64_cortex-a53.ipk.
Installing uci (2023.08.10~5781664d-r1) to root...
Copying /builder/dl/uci_2023.08.10~5781664d-r1_aarch64_cortex-a53.ipk.
Package uclient-fetch (2024.04.19~e8780fa7-r1) installed in root is up to date.
Installing urandom-seed (3) to root...
Copying /builder/dl/urandom-seed_3_aarch64_cortex-a53.ipk.
Installing urngd (2023.11.01~44365eb1-r1) to root...
Copying /builder/dl/urngd_2023.11.01~44365eb1-r1_aarch64_cortex-a53.ipk.
Installing usbutils (017-r1) to root...
Copying /builder/dl/usbutils_017-r1_aarch64_cortex-a53.ipk.
Installing libatomic1 (13.3.0-r4) to root...
Copying /builder/dl/libatomic1_13.3.0-r4_aarch64_cortex-a53.ipk.
Installing libusb-1.0-0 (1.0.27-r1) to root...
Copying /builder/dl/libusb-1.0-0_1.0.27-r1_aarch64_cortex-a53.ipk.
Installing libevdev (1.13.1-r1) to root...
Copying /builder/dl/libevdev_1.13.1-r1_aarch64_cortex-a53.ipk.
Installing libudev-zero (1.0.3-r1) to root...
Copying /builder/dl/libudev-zero_1.0.3-r1_aarch64_cortex-a53.ipk.
Installing usbids (0.387-r1) to root...
Copying /builder/dl/usbids_0.387-r1_aarch64_cortex-a53.ipk.
Installing wpad-mbedtls (2024.09.15~5ace39b0-r1) to root...
Copying /builder/dl/wpad-mbedtls_2024.09.15~5ace39b0-r1_aarch64_cortex-a53.ipk.
Installing ucode-mod-uloop (2024.07.22~b610860d-r3) to root...
Copying /builder/dl/ucode-mod-uloop_2024.07.22~b610860d-r3_aarch64_cortex-a53.ipk.
Package base-files (1627~f9f2dd1d93) installed in root is up to date.
Package busybox (1.36.1-r2) installed in root is up to date.
Package ca-bundle (20240203-r1) installed in root is up to date.
Package dnsmasq (2.90-r3) installed in root is up to date.
Package dropbear (2024.85-r1) installed in root is up to date.
Package firewall4 (2024.05.21~4c01d1eb-r1) installed in root is up to date.
Package fitblk (2) installed in root is up to date.
Package kernel (6.6.56~fd58c1f219e8bdd2c94a26335acdfb0a-r1) installed in root is up to date.
Package kmod-crypto-hw-safexcel (6.6.56-r1) installed in root is up to date.
Package kmod-gpio-button-hotplug (6.6.56-r5) installed in root is up to date.
Package kmod-leds-gpio (6.6.56-r1) installed in root is up to date.
Package kmod-mt7915e (6.6.56.2024.10.11.1~ecca0e77-r1) installed in root is up to date.
Package kmod-mt7981-firmware (6.6.56.2024.10.11.1~ecca0e77-r1) installed in root is up to date.
Package kmod-nft-offload (6.6.56-r1) installed in root is up to date.
Package kmod-nvme (6.6.56-r1) installed in root is up to date.
Package kmod-phy-airoha-en8811h (6.6.56-r1) installed in root is up to date.
Package kmod-phy-aquantia (6.6.56-r1) installed in root is up to date.
Package kmod-rtc-pcf8563 (6.6.56-r1) installed in root is up to date.
Package kmod-usb3 (6.6.56-r1) installed in root is up to date.
Package libustream-mbedtls20201210 (2024.07.28~99bd3d2b-r1) installed in root is up to date.
Package logd (2024.04.26~85f10530-r1) installed in root is up to date.
Package mt7981-wo-firmware (20240909-r1) installed in root is up to date.
Package mtd (26) installed in root is up to date.
Package odhcp6c (2023.05.12~bcd28363-r20) installed in root is up to date.
Package odhcpd-ipv6only (2024.05.08~a2988231-r1) installed in root is up to date.
Package opkg (2024.10.16~38eccbb1-r1) installed in root is up to date.
Package ppp (2.5.1-r1) installed in root is up to date.
Package ppp-mod-pppoe (2.5.1-r1) installed in root is up to date.
Package procd (2024.07.07~f230c117-r1) installed in root is up to date.
Package procd-seccomp (2024.07.07~f230c117-r1) installed in root is up to date.
Package procd-ujail (2024.07.07~f230c117-r1) installed in root is up to date.
Package uboot-envtools (2024.07-r1) installed in root is up to date.
Package uci (2023.08.10~5781664d-r1) installed in root is up to date.
Package uclient-fetch (2024.04.19~e8780fa7-r1) installed in root is up to date.
Package urandom-seed (3) installed in root is up to date.
Package urngd (2023.11.01~44365eb1-r1) installed in root is up to date.
Configuring kmod-scsi-core.
Configuring kmod-nls-base.
Configuring kmod-usb-core.
Configuring kmod-usb-storage.
Configuring libnl-tiny1.
Configuring libubox20240329.
Configuring libuci20130104.
Configuring libubus20231128.
Configuring libiwinfo-data.
Configuring libiwinfo20230701.
Configuring iwinfo.
Configuring liblucihttp0.
Configuring cgi-io.
Configuring libjson-c5.
Configuring libblobmsg-json20240329.
Configuring rpcd.
Configuring rpcd-mod-file.
Configuring rpcd-mod-luci.
Configuring libucode20230711.
Configuring rpcd-mod-ucode.
Configuring ucode.
Configuring ucode-mod-fs.
Configuring ucode-mod-uci.
Configuring ucode-mod-ubus.
Configuring ucode-mod-math.
Configuring ucode-mod-html.
Configuring liblucihttp-ucode.
Configuring luci-base.
Configuring luci-lib-chartjs.
Configuring zlib.
Configuring kmod-nfnetlink.
Configuring kmod-nf-conntrack.
Configuring kmod-nf-conntrack-netlink.
Configuring nlbwmon.
Configuring luci-app-nlbwmon.
Configuring libpthread.
Configuring librt.
Configuring libuuid1.
Configuring libf2fs6.
Configuring mkf2fs.
Configuring kmod-crypto-gf128.
Configuring libuclient20201210.
Configuring uclient-fetch.
Configuring opkg.
Configuring libevdev.
Configuring luci-app-opkg.
Configuring ubusd.
Configuring ubus.
Configuring iw.
Configuring kmod-crypto-hash.
Configuring kmod-crypto-null.
Configuring kmod-crypto-aead.
Configuring kmod-crypto-manager.
Configuring busybox.
Configuring f2fsck.
Configuring f2fs-tools.
Configuring kmod-crypto-hmac.
Configuring kmod-crypto-sha512.
Configuring kmod-crypto-sha3.
Configuring kmod-crypto-rng.
Configuring kmod-crypto-geniv.
Configuring kmod-crypto-seqiv.
Configuring kmod-crypto-ctr.
Configuring kmod-nf-reject.
Configuring kmod-nf-reject6.
Configuring kmod-nf-conntrack6.
Configuring kmod-nf-nat.
Configuring kmod-nf-log.
Configuring kmod-nf-log6.
Configuring kmod-crypto-crc32c.
Configuring kmod-lib-crc32c.
Configuring kmod-nft-core.
Configuring kmod-nft-fib.
Configuring libcares.
Configuring libmbedtls21.
Configuring libnghttp2-14.
Configuring ca-bundle.
Configuring libcurl4.
Configuring libev.
Configuring jsonfilter.
Configuring resolveip.
Configuring https-dns-proxy.
Configuring luci-app-https-dns-proxy.
Configuring kmod-crypto-md5.
Configuring luci-mod-system.
Configuring kmod-nf-flow.
Configuring kmod-usb-xhci-hcd.
Configuring kmod-usb-xhci-mtk.
Configuring kmod-lib-crc-ccitt.
Configuring ucode-mod-uloop.
Configuring getrandom.
Configuring kmod-libphy.
Configuring kmod-hwmon-core.
Configuring kmod-phy-aquantia.
Configuring luci-theme-bootstrap.
Configuring libatomic1.
Configuring libusb-1.0-0.
Configuring kmod-slhc.
Configuring kmod-ppp.
Configuring kmod-pppox.
Configuring kmod-pppoe.
Configuring procd-ujail.
Configuring jshn.
Configuring libudebug.
Configuring netifd.
Configuring usign.
Configuring openwrt-keyring.
Configuring ubi-utils.
Configuring ubox.
Configuring fstools.
Configuring fwtool.
Configuring base-files.
Configuring kmod-lib-crc16.
Configuring libustream-mbedtls20201210.
Configuring fitblk.
Configuring kmod-nft-nat.
Configuring kmod-nft-offload.
Configuring libmnl0.
Configuring libnftnl11.
Configuring jansson4.
Configuring nftables-json.
Configuring firewall4.
Configuring uboot-envtools.
Configuring dnsmasq.
Configuring libjson-script20240329.
Configuring procd.
Configuring airoha-en8811h-firmware.
Configuring libudev-zero.
Configuring usbids.
Configuring usbutils.
Configuring kmod-crypto-sha256.
Configuring kmod-nvme.
Configuring kmod-crypto-acompress.
Configuring kmod-i2c-core.
Configuring kmod-rtc-pcf8563.
Configuring rpcd-mod-iwinfo.
Configuring luci-mod-status.
Configuring usteer.
Configuring luci-app-usteer.
Configuring ucode-mod-nl80211.
Configuring ucode-mod-rtnl.
Configuring wifi-scripts.
Configuring wireless-regdb.
Configuring kmod-cfg80211.
Configuring kmod-crypto-cmac.
Configuring kmod-crypto-ccm.
Configuring kmod-crypto-ghash.
Configuring kmod-crypto-gcm.
Configuring hostapd-common.
Configuring kmod-mac80211.
Configuring kmod-mt76-core.
Configuring kmod-mt76-connac.
Configuring block-mount.
Configuring kmod-usb3.
Configuring kmod-fs-exfat.
Configuring luci-app-firewall.
Configuring kmod-thermal.
Configuring libblkid1.
Configuring odhcp6c.
Configuring uci.
Configuring kmod-fs-ext4.
Configuring dropbear.
Configuring curl.
Configuring mtd.
Configuring odhcpd-ipv6only.
Configuring procd-seccomp.
Configuring eip197-mini-firmware.
Configuring kmod-crypto-authenc.
Configuring kmod-crypto-des.
Configuring kmod-crypto-sha1.
Configuring kmod-crypto-hw-safexcel.
Configuring kmod-mt7915e.
Configuring urandom-seed.
Configuring luci-proto-ppp.
Configuring kmod-fs-autofs4.
Configuring blockd.
Configuring libcomerr0.
Configuring ppp.
Configuring luci-mod-network.
Configuring luci-mod-admin-full.
Configuring kmod-leds-gpio.
Configuring kmod-gpio-button-hotplug.
Configuring logd.
Configuring luci-proto-ipv6.
Configuring rpcd-mod-rpcsys.
Configuring attendedsysupgrade-common.
Configuring luci-app-attendedsysupgrade.
Configuring libss2.
Configuring libext2fs2.
Configuring libe2p2.
Configuring e2fsprogs.
Configuring kmod-crypto-crc32.
Configuring kmod-fs-f2fs.
Configuring wpad-mbedtls.
Configuring mt7981-wo-firmware.
Configuring uhttpd.
Configuring rpcd-mod-rrdns.
Configuring uhttpd-mod-ubus.
Configuring luci-light.
Configuring luci.
Configuring libstdcpp6.
Configuring kmod-mt7981-firmware.
Configuring kmod-lib-lzo.
Configuring kmod-phy-airoha-en8811h.
Configuring gdisk.
Configuring urngd.
Configuring ppp-mod-pppoe.

Finalizing root filesystem...
Enabling blockd
Enabling boot
Enabling bootcount
Enabling cron
Enabling dnsmasq
Enabling done
Enabling dropbear
Enabling firewall
Enabling fstab
Enabling gpio_switch
Enabling https-dns-proxy
Enabling led
Enabling log
Enabling network
Enabling nlbwmon
Enabling odhcpd
Enabling packet_steering
Enabling radius
Enabling rpcd
Enabling sysctl
Enabling sysfixtime
Enabling sysntpd
Enabling system
Enabling ucitrack
Enabling uhttpd
Enabling umount
Enabling urandom_seed
Enabling urngd
Enabling usteer
Enabling wpad

Building images...
Parallel mksquashfs: Using 16 processors
Creating 4.0 filesystem on /builder/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/root.squashfs, block size 262144.

[|                                                             ]    0/1476   0%
[=============================================================|] 1476/1476 100%

Exportable Squashfs 4.0 filesystem, xz compressed, data block size 262144
	compressed data, compressed metadata, compressed fragments,
	no xattrs, compressed ids
	duplicates are removed
Filesystem size 7010.11 Kbytes (6.85 Mbytes)
	22.53% of uncompressed filesystem size (31109.39 Kbytes)
Inode table size 12280 bytes (11.99 Kbytes)
	20.23% of uncompressed inode table size (60711 bytes)
Directory table size 17610 bytes (17.20 Kbytes)
	43.02% of uncompressed directory table size (40938 bytes)
Number of duplicate files found 224
Number of inodes 1829
Number of files 1431
Number of fragments 73
Number of symbolic links 266
Number of device nodes 1
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 131
Number of hard-links 0
Number of ids (unique uids + gids) 1
Number of uids 1
	unknown (0)
Number of gids 1
	unknown (0)
FIT description: ARM64 OpenWrt FIT (Flattened Image Tree)
Created:         Fri Oct 18 06:03:34 2024
 Image 0 (kernel-1)
  Description:  ARM64 OpenWrt Linux-6.6.56
  Created:      Fri Oct 18 06:03:34 2024
  Type:         Kernel Image
  Compression:  gzip compressed
  Data Size:    5912662 Bytes = 5774.08 KiB = 5.64 MiB
  Architecture: AArch64
  OS:           Linux
  Load Address: 0x44000000
  Entry Point:  0x44000000
  Hash algo:    crc32
  Hash value:   f324408f
  Hash algo:    sha1
  Hash value:   c5393431775f513346fbd1c2883b493469c74397
 Image 1 (fdt-1)
  Description:  ARM64 OpenWrt openwrt_one device tree blob
  Created:      Fri Oct 18 06:03:34 2024
  Type:         Flat Device Tree
  Compression:  uncompressed
  Data Size:    29860 Bytes = 29.16 KiB = 0.03 MiB
  Architecture: AArch64
  Load Address: 0x43f00000
  Hash algo:    crc32
  Hash value:   2a2a56a0
  Hash algo:    sha1
  Hash value:   dc150890e03bd079d7cd10711ba29554a0c10a27
 Image 2 (rootfs-1)
  Description:  ARM64 OpenWrt openwrt_one rootfs
  Created:      Fri Oct 18 06:03:34 2024
  Type:         Filesystem Image
  Compression:  uncompressed
  Data Size:    7180288 Bytes = 7012.00 KiB = 6.85 MiB
  Hash algo:    crc32
  Hash value:   f2db0518
  Hash algo:    sha1
  Hash value:   36f2f674c6072595f56f5f2c87ea9c8501462c38
 Default Configuration: 'config-1'
 Configuration 0 (config-1)
  Description:  OpenWrt openwrt_one
  Kernel:       kernel-1
  FDT:          fdt-1
  Loadables:    rootfs-1
[ubootenv]
mode=ubi
vol_id=0
vol_type=dynamic
vol_name=ubootenv
vol_size=1MiB
[ubootenv2]
mode=ubi
vol_id=1
vol_type=dynamic
vol_name=ubootenv2
vol_size=1MiB
[fip]
mode=ubi
vol_id=2
vol_type=static
vol_name=fip
image=/builder/staging_dir/target-aarch64_cortex-a53_musl/image/mt7981_openwrt_one-snand-u-boot.fip
vol_size=961536
[recovery]
mode=ubi
vol_id=3
vol_type=static
vol_name=recovery
image=/builder/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb
vol_size=0
[fit]
mode=ubi
vol_id=4
vol_type=dynamic
vol_name=fit
image=/builder/8c140f24fa43c4cbbab2e10082c3c27b/openwrt-861f3e625922-mediatek-filogic-openwrt_one-squashfs-sysupgrade.itb
vol_size=13239296

STDERR:

Package list missing or not up-to-date, generating it.

Building package index...
Downloading https://downloads.openwrt.org/snapshots/targets/mediatek/filogic/packages/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_core
Downloading https://downloads.openwrt.org/snapshots/targets/mediatek/filogic/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/base/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_base
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/base/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/targets/mediatek/filogic/kmods/6.6.56-1-fd58c1f219e8bdd2c94a26335acdfb0a/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_kmods
Downloading https://downloads.openwrt.org/snapshots/targets/mediatek/filogic/kmods/6.6.56-1-fd58c1f219e8bdd2c94a26335acdfb0a/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/luci/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_luci
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/luci/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/packages/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_packages
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/packages/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/routing/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_routing
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/routing/Packages.sig
Signature check passed.
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/telephony/Packages.gz
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/openwrt_telephony
Downloading https://downloads.openwrt.org/snapshots/packages/aarch64_cortex-a53/telephony/Packages.sig
Signature check passed.
Downloading file:packages/Packages
Updated list of available packages in /builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/../../../../builder/dl/imagebuilder
Downloading file:packages/Packages.sig
Signature check passed.
/builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/etc/init.d/usteer: line 10: /lib/functions/network.sh: No such file or directory
/builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/etc/init.d/usteer: line 11: /usr/share/libubox/jshn.sh: No such file or directory
/builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/etc/init.d/usteer: line 12: /lib/functions.sh: No such file or directory
./etc/init.d/usteer: line 10: /lib/functions/network.sh: No such file or directory
./etc/init.d/usteer: line 11: /usr/share/libubox/jshn.sh: No such file or directory
./etc/init.d/usteer: line 12: /lib/functions.sh: No such file or directory
Pseudo file "dev" exists in source filesystem "/builder/build_dir/target-aarch64_cortex-a53_musl/root-mediatek/dev".
Ignoring, exclude it (-e/-ef) to override.
11548+1 records in
11548+1 records out
5912662 bytes (5.9 MB, 5.6 MiB) copied, 0.0134615 s, 439 MB/s
1752+1 records in
1753+0 records out
7180288 bytes (7.2 MB, 6.8 MiB) copied, 0.00728729 s, 985 MB/s
padding image to 00ca0000
stat: cannot statx '/builder/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb': No such file or directory
ubinize: error!: cannot stat "/builder/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb" referred from section "recovery"
         error 2 (No such file or directory)
make[3]: *** [Makefile:43: /builder/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/tmp/openwrt-861f3e625922-mediatek-filogic-openwrt_one-factory.ubi] Error 2
make[2]: *** [Makefile:247: build_image] Error 2
make[1]: *** [Makefile:153: _call_image] Error 2
make: *** [Makefile:310: image] Error 2

```

--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -1090,7 +1090,9 @@ define Device/openwrt_one
   UBINIZE_OPTS := -E 5
   BLOCKSIZE := 128k
   PAGESIZE := 2048
-  UBINIZE_PARTS := fip=:$(STAGING_DIR_IMAGE)/mt7981_openwrt_one-snand-u-boot.fip recovery=:$(KDIR)/tmp/openwrt-mediatek-filogic-openwrt_one-initramfs.itb \
+  UBINIZE_PARTS := fip=:$(STAGING_DIR_IMAGE)/mt7981_openwrt_one-snand-u-boot.fip \
+		   $(if $(IB),recovery=:$(STAGING_DIR_IMAGE)/mediatek-filogic-openwrt_one-initramfs.itb,\
+			      recovery=:$(KDIR)/tmp/$$(KERNEL_INITRAMFS_IMAGE)) \
 		   $(if $(wildcard $(TOPDIR)/openwrt-mediatek-filogic-openwrt_one-calibration.itb), calibration=:$(TOPDIR)/openwrt-mediatek-filogic-openwrt_one-calibration.itb)
 endef
 TARGET_DEVICES += openwrt_one

